src/CommonItilObject_Item.php

```diff
@@ -1578,4 +1509,213 @@ public static function dropdownAllDevices(
         }
         return $rand;
     }
+
+
+    /**
+     * Return used items for a commonitilobject
+     *
+     * @param integer $itilobject_id
+     * @param string  $itilobject_type
+     *
+     * @return array
+     */
+    public static function getUsedItems($itilobject_id, $itilobject_type)
+    {
+
+        $data = getAllDataFromTable(static::getTable(), [getForeignKeyFieldForItemType($itilobject_type) => $itilobject_id]);
+        $used = [];
+        if (!empty($data)) {
+            foreach ($data as $val) {
+                $used[$val['itemtype']][] = $val['items_id'];
+            }
+        }
+
+        return $used;
+    }
+
+
+    public static function showItemToAdd($itemtype, $items_id, $options)
+    {
+        $params = [
+            'rand'      => mt_rand(),
+            'delete'    => true,
+            'visible'   => true,
+            'kblink'    => true
+        ];
+
+        foreach ($options as $key => $val) {
+            $params[$key] = $val;
+        }
+
+        $result = "";
+
+        if ($item = getItemForItemtype($itemtype)) {
+            if ($params['visible']) {
+                $item->getFromDB($items_id);
+                $result =  "<div id='{$itemtype}_$items_id'>";
+                $result .= $item->getTypeName(1) . " : " . $item->getLink(['comments' => true]);
+                $result .= Html::hidden("items_id[$itemtype][$items_id]", ['value' => $items_id]);
+                if ($params['delete']) {
+                    $result .= " <i class='fas fa-times-circle pointer' onclick=\"itemAction" . $params['rand'] . "('delete', '$itemtype', '$items_id');\"></i>";
```

Comment:
```suggestion
                    $result .= " <i class='ti ti-circle-x pointer' onclick=\"itemAction" . $params['rand'] . "('delete', '$itemtype', '$items_id');\"></i>";
```
This should probably be a button too.

Reply:
done